Makefile

```diff
@@ -0,0 +1,7 @@
+generate-grpc-code:
```

Comment:
How about add some instructions on how to run the command to re-generate or refresh those code?

Reply:
Good call! 

Should we consider adding it on `CONTRIBUTING.md` instead? or what do you think

Reply:
Yeah I think that sounds great

Reply:
Done. Fully inspired from the cadence server one :)